bin/initPolykey.ts

```diff
@@ -157,7 +159,19 @@ async function initPolyKey(): Promise<PolyKey> {
       const keyGenerationName = configStore.get('keyGenerationName')
       const keyGenerationEmail = configStore.get('keyGenerationEmail')
       const keyGenerationPassphrase = configStore.get('keyGenerationPassphrase')
-      await keyManager.generateKeyPair(keyGenerationName, keyGenerationEmail, keyGenerationPassphrase)
+
+      // Validate passphrase
+      const passValidation = zxcvbn(keyGenerationPassphrase)
+      // The following is an arbitrary delineation of desirable scores
+      if (passValidation.score < 2) {
+        pkLogger('passphrase score for new keypair is below 2!', PKMessageType.WARNING)
```

Comment:
Should we allow an opportunity for the user to redo this? Of course we don't want to limit the user in this case. But in GUI, it's usually a UI info message. For CLI, we just do a warning.

Reply:
yeah since this is now only as a warning in the CLI, and not actually in the polykey library itself, we can redo it in the polykey GUI